docs/BR.md

```diff
@@ -3151,7 +3151,7 @@ A full and complete CRL is a list of all revoked certificates issued by the CA f
 
 A partitioned CRL (sometimes referred to as a "sharded CRL") is a list of revoked certificates issued by the CA for any and all reasons constrained to a specific scope (e.g., temporal sharding).
 
-Minimally, CAs MUST issue either a "full and complete" CRL - or "partitioned" CRLs.
+Minimally, CAs MUST issue either a "full and complete" CRL - or "partitioned" CRLs. CAs MUST NOT issue indirect CRLs (i.e., the issuer of the CRL is not the issuer of all Certificates that are included in the scope of the CRL).
```

Comment:
I *think* this is a natural consequence of requiring `cRLSign` to be asserted in the keyUsage extension of both roots and intermediates.

Reply:
Accepting proposed change. 

(thank you!)